src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/client/util/HomeOfficeApiUtil.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.client.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.AsylumCaseDefinition;
+
+@Service
+public class HomeOfficeApiUtil {
+
+    public HttpHeaders getHomeOfficeHeader() {
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
+
+        return headers;
+    }
+
+    public Map<String, Object> createMessageHeader() {
+        Map<String, Object> messageHeader = new HashMap<>();
+        messageHeader.put("consumer", createConsumer());
+        messageHeader.put("correlationId", "ABC2344BCED2234EA");
+        messageHeader.put("eventDateTime", "2020-05-19T17:32:28Z");
+
+        return messageHeader;
+    }
+
+    Map<String, String> createConsumer() {
+        return ImmutableMap.of("code", "HMCTS", "description", "HM Courts and Tribunal Service");
+    }
+
+    public AsylumCase extractHomeOfficeData(String response) throws JsonProcessingException {
+        AsylumCase homeOfficeData = new AsylumCase();
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Again we should be modelling this response properly in your next task with jackson.
We should also consider how we are going to put all the data in the CCD case data.  Should be a distinction between what is home-office-case-data and hmcts/CCD case-data

Reply:
Will work on it next

Reply:
I am not using the response so this will be the functionality to develop later